Package.swift

```diff
@@ -1,19 +1,28 @@
-//    Copyright 2016 Swift Studies
-//
-//    Licensed under the Apache License, Version 2.0 (the "License");
-//    you may not use this file except in compliance with the License.
-//    You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-//    Unless required by applicable law or agreed to in writing, software
-//    distributed under the License is distributed on an "AS IS" BASIS,
-//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//    See the License for the specific language governing permissions and
-//    limitations under the License.
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
-	name : "Duration"
-)
\ No newline at end of file
+    name: "Duration",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "Duration",
+            targets: ["Duration"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "Duration",
+            dependencies: []),
+        .testTarget(
+            name: "DurationTests",
+            dependencies: ["Duration"]),
+    ]
```

Comment:
Unless you're going to write some useful tests, I recommend removing the test target and the dummy test files that were generated.

Reply:
OK, I remove it.

Reply:
Sorry I make mistake in my post, `travis.yml` is not changed. Please ignore it.